proto/ibc/core/commitment/v1/commitment.proto

```diff
@@ -4,7 +4,7 @@ package ibc.core.commitment.v1;
 option go_package = "github.com/cosmos/cosmos-sdk/x/ibc/core/23-commitment/types";
 
 import "gogoproto/gogo.proto";
-import "confio/proofs.proto";
+import "proofs.proto";
```

Comment:
This is unfortunate (ref: https://github.com/cosmos/cosmos-sdk/pull/9253#issuecomment-832812453). In 07cf792c7c24d351abed6ebed40b83fe32c7ef2d I moved proofs.proto to the third_party/proto root folder.

Does this workaround work for you @fdymylja?

Is the correct way of solving this to create a PR on https://github.com/confio/ics23/ to move the `proofs.proto` file there into a `confio` subfolder?

Reply:
An alternative is just to not backport this fix...

Reply:
Discussed with @robert-zaremba, let's just not backport this PR, and ask people to use 0.43 if they want server reflection.